src/CartFee.php

```diff
@@ -23,12 +24,14 @@ class CartFee
      *
      * @param $amount
      * @param $taxable
+     * @param $tax
      * @param array $options
      */
-    public function __construct($amount, $taxable, $options = [])
+    public function __construct($amount, $taxable = false, $tax = 0, $options = [])
```

Comment:
you can set it as a public var above and set a default value


public $tax = 0;